src/Effects/Tremolo.js

```diff
@@ -0,0 +1,83 @@
+Pizzicato.Effects.Tremolo = function(options) {
+
+	this.options = {};
+	options = options || this.options;
+
+	var defaults = {
```

Comment:
A couple ideas for future parameters would be:

- Tremolo depth: which could be implemented with another gain node to scale the signal, and
- Wave shape: currently we have a sinusoidal wave coming from the oscillator that controls the lfo, eventually we could add different shapes. 

These two additions would make the API interface close to the pedal [Boss TR-2](https://www.youtube.com/watch?v=e8xkukid_2o).